packages/office-ui-fabric-react/src/components/DatePicker/__snapshots__/DatePicker.test.tsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`DatePicker renders default DatePicker correctly 1`] = `
 <div
-  className="ms-DatePicker"
+  className={undefined}
```

Comment:
this should not be undefined

Reply:
Per Teams, `DatePicker.test.tsx` snapshot test should be modified to use `DatePicker` instead of `DatePickerBase`.